uml4net.xmireader/Readers/Packages/PackageReader.cs

```diff
@@ -156,7 +156,8 @@ public override IPackage Read(XmlReader xmlReader)
                                 this.ReadPackagedElements(package, xmlReader);
                                 break;
                             default:
-                                throw new NotImplementedException($"PackageReader: {xmlReader.LocalName}");
+                                this.Logger.LogInformation("PackageReader. {type} not yet implemented", xmlReader.LocalName);
```

Comment:
i prefer we maintain the exception, otherwise we will not find out what is mssing. Please put the exception back and add a case to switch statement for the missing type